docs/metasploit-framework.wiki/How-to-use-Railgun-for-Windows-post-exploitation.md

```diff
@@ -72,70 +74,81 @@ msf exploit(handler) > run
 [*] Meterpreter session 1 opened (192.168.1.64:4444 -> 192.168.1.106:55148) at 2014-07-30 19:49:35 -0500
 
 meterpreter > irb
-[*] Starting IRB shell
-[*] The 'client' variable holds the meterpreter client
+[*] Starting IRB shell...
+[*] You are in the "client" (session) object
 
 >>
```

Comment:
What are your thoughts on adding an extra intermediate assignment here for clarity to the user?

```suggestion
>> session = self
=> #<Session:meterpreter 192.168.1.106:5514 (192.168.1.64) "user @ 192.168.1.64">
>>
```

That way the later examples can continue to use the long-form `session.railgun` for explicitness 🤔

Reply:
Well on the one hand if we do that, it'll be another step that users may forget to execute before they can copy-paste examples. On the other, if we did access the methods from `session`, then users could copy and paste the examples into modules.

Reply:
Either way I guess it will require users to read the documentation 😢